workflow-service/src/main/java/com/redhat/parodos/workflow/execution/controller/WorkFlowController.java

```diff
@@ -113,6 +113,25 @@ public void updateWorkFlowCheckerTaskStatus(@PathVariable UUID workFlowExecution
 				workFlowCheckerTaskRequestDTO.getStatus());
 	}
 
+	@Operation(summary = "Restart a workflow execution with same parameters")
+	@ApiResponses(value = {
+			@ApiResponse(responseCode = "202", description = "Accepted",
+					content = { @Content(mediaType = "application/json",
+							schema = @Schema(implementation = WorkFlowExecutionResponseDTO.class)) }),
+			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content),
+			@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content) })
+	@PostMapping("/{workFlowExecutionId}")
+	public ResponseEntity<WorkFlowExecutionResponseDTO> restartWorkflow(@PathVariable UUID workFlowExecutionId) {
```

Comment:
pls rename to restartWorkFlow (seems like the operationId in the API uses WorkFlow)